README.md

```diff
@@ -45,12 +45,13 @@ And something like this would render an input that would update the underlying v
 
 ## Usage
 
-The entry point to a UI powered by ember-exclaim is the `{{exclaim-ui}}` component. It expects up to five arguments:
+The entry point to a UI powered by ember-exclaim is the `{{exclaim-ui}}` component. It accepts the following properties:
  - `ui`: an object containing configuration for the UI that should be rendered
  - `env`: a hash whose keys will be bindable from the `ui` config, to be read from and written to
  - `componentMap`: a mapping of component names in the `ui` config to information about their backing Ember components
- - `resolveMeta(path)`: an optional action that will be invoked if a component calls `env.metaFor(...)`
+ - `resolveFieldMeta(path)`: an optional action that will be invoked if a component calls `env.metaForField(...)`
  - `onChange(envKeyOfChangedValue)`: an optional action that will be invoked when a value in the `env` changes
+ - `wrapper`: an optional component or component name string that will wrap every rendered component in your UI configuration. The `wrapper` component will receive the unwrapped `ComponentSpec` as `spec` ([more on `ComponentSpec` here](addon/-private/GLOSSARY.md)), the `Environment` as `env` and the component's resolved `config`.
```

Comment:
Worth adding a note in the glossary about `ComponentSpec`s also having implementation-defined metadata?

Reply:
yeah I'll put that in